_component_design/lists.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Lists
+parent: Content Areas
+layout: component
+category: Components
+usage: >
+  Lists help break up large blocks of text, make complex articles and blog posts easier to grasp, and make key information stand out.
+preview-image: preview-images/list.svg
+resource: true
+status: stable
+last-modified: 2017-11-16
+---
+
+<section class="static-section"  markdown="1">
```

Comment:
Are we skipping the TOCs now?

Also, there's two section starters here, one before and one after the Lists heading. Please remove one.

Reply:
Nope,  bad copy pasta on my part. Added the `{% include toc.html %}` and removed the empty section